src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin.csproj

```diff
@@ -1,6 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" 
```

Comment:
Revert this change.

Reply:
visual studio combined the two lines into one, any issues with this ?

Reply:
not really.  Just won't look like the other csproj files.

Reply:
reverted to match other files.